src/docs/ganache/reference/cli-options.md

```diff
@@ -0,0 +1,917 @@
+    ```
+
+=== "Example"
+
+    ```bash
+    --fork.blockNumber=182354
+    ```
+
+Block number to fork from. Defaults to the latest block.
+
+**`--fork.preLatestConfirmations`**
+
+=== "Syntax"
+    ```bash
+    --fork.preLatestConfirmations=<NUMBER>
+    ```
+
+=== "Example"
+
+    ```bash
+    --fork.preLatestConfirmations=8
+    ```
+
+When the `--fork.blockNumber` is set to `latest` (default), this option specifies the number of blocks
+before the remote node's latest block to fork from.
+
+The default is `5`.
+
+**`--fork.username`**
+
+=== "Syntax"
+    ```bash
+    --fork.username=<STRING>
+    ```
+
+=== "Example"
+
+    ```bash
+    --fork.username="JohnDoe"
+    ```
+
+Username for basic authentication. Does not require setting `--fork.password`.
+When combined with `--fork.password`, the shorthand is `{ headers: { "Authorization": "Basic {ENCODED-BASIC-HEADER}" } }`
+
+If `--fork.headers` specifies an authorization header, it will be inserted after the Basic token.
+
+**`--fork.password`**
+
+=== "Syntax"
+    ```bash
+    --fork.password=<STRING>
+    ```
+
+=== "Example"
+
+    ```bash
+    --fork.password="changeme"
+    ```
+
+Password for basic authentication. Does not require setting `--fork.username`.
+When combined with `--fork.username`, the shorthand is `{ headers: { "Authorization": "Basic {ENCODED-BASIC-HEADER}" } }`
+
+If the `--fork.headers` specifies an authorization header, it will be inserted after the Basic token.
+
+**`--fork.jwt`**
+
+=== "Syntax"
+    ```bash
+    --fork.jwt=<STRING>
+    ```
+
+=== "Example"
+
+    ```bash
+    --fork.jwt="eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiJ9.eyJwZXJtaXNzaW9ucyI6Iio6KiIsInByaXZhY3lQdWJsaWNLZXkiOiIyVUtIM1ZKVGhrT29Lc2tyTEZwd294Q25uUkFSeW9iVjFiRWRnc2VGSFRzPSIsImV4cCI6IjE2MDA4OTk5OTkwMDIiLCJpYXQiOjE2MzkxNTc2Mjd9.FGf-FmfDQlIPCRDGmNnsHZWlwrUr69d7AIDqQrIrUrSJLiwGpR3NCUhVHIDMpQvDHQYf-sFMZTYvZGrvztYRuBKWMbTfIZKN74onzNJbFIPBVQuUX2HMXmI4VQ3UFB11LShiUJHKLna13qdbqfbgJIO3HetxJhJQxTiwtixfHwyPXl-Nx8HbQy_AWH58lLAUeaoLzN7QIA9kborthBpvfK9C7Sv1lXT1cdCDC4oRKBoiMg2RWFZtGtxFsnWyloangwbhCB6Bc_elqY5nd9WkF4ix95xsP_HgBcouy1sDw6jxn5_LveX53H8owczVWP6S1e6hv6hq2fs6YkSntKMK2g"
+    ```
+
+Encoded JSON Web Token (JWT) used for authenticating to servers.
+
+The shorthand is `{ headers: { "Authorization": "Bearer {YOUR-ENCODED-JWT}" } }`
+
+If the `--fork.headers` option specifies an authorization header, it will be inserted after the JWT Bearer token.
+
+**`--fork.userAgent`**
+
+=== "Syntax"
+    ```bash
+    --fork.userAgent=<STRING>
+    ```
+
+=== "Example"
+
+    ```bash
+    --fork.userAgent="Ganache/7.0.0-beta.0 (https://www.trufflesuite.com/ganache; ganache<at>trufflesuite.com)]"
+    ```
+
+The User-Agent header sent to the fork on each request.
+
+Sent as `Api-User-Agent` when used in the browser. Is overridden by a `User-Agent` defined in the `--fork.headers`
+option, if provided.
+
+The default is `Ganache/7.0.0-beta.0 (https://www.trufflesuite.com/ganache; ganache<at>trufflesuite.com)]`
```

Comment:
Aware this is coming from the README, but it feels a little funky (cc @davidmurdoch). Same goes for the example on [L745](https://github.com/trufflesuite/trufflesuite.com/pull/1312/files#diff-f929cf97d16f01f1875b744299cf3cd202190ad45781da6d4cffd10aed95bbbbR745).

Reply:
The default value updates based off of the version of Ganache being run, so maybe we could update that accordingly? Also, there's an extra `]` at the end which could have been making it look more funky
```suggestion
The default is `Ganache/VERSION (https://www.trufflesuite.com/ganache; ganache<at>trufflesuite.com)`
```